### DISCUSSION TITLE
Are the functions of the charts page in the GDS finished yet?

### DISCUSSION THREAD

**Discussion by kitty99099** [nasa/fprime#2963] [Q&A] [answered]
Hello everyone.

After finishing all of my project. I would like to use the charts to watch the data in a visualized way. I found the chart page can satisfy that. I can definetly use the page to show my IMU xyz data respectively as below. Also, I can zoom in/out like the `help` botton said.

![image](https://github.com/user-attachments/assets/9496f9b1-ef01-4682-8f59-094ccf466a61)

However, the `save` botton seemed not work correctly. It only showed the names but no data. The content of the download file is shown as below.

![image](https://github.com/user-attachments/assets/c6070ab5-85b9-47e2-98f1-df0097855c21)

I tried to search from the [Guide to F' GDS](https://nasa.github.io/fprime/UsersGuide/gds/gds-introduction.html), but it skip the introduction of charts page.

I'm not sure if I did something wrong or it is because the function of the charts page in the GDS not finished yet. I will be grateful if anyone know this problem.

**Comment by thomas-bc** [comment]
@LeStarch would you know about this?

**Comment by LeStarch** [comment, marked as answer]
The save button here may be a touch misleading.  It means "Save this view" such that the same view can be loaded up again in the future.  It is not saving the data, but the list of charts set up so that this view can be easily reconstructed.

**Comment by kitty99099** [comment]
Thank both of you for your help and explaination. @thomas-bc @LeStarch
